### PR TITLE
Return EOA address on WalletConnect

### LINKED ISSUE #1075: Return EOA address on WalletConnect
### Current behavior
Wallet Connect integration is returning the MTW wallet instead of EOA. We want to use the EOA.

### Desired behavior
WC integration returns EOA

### PR DESCRIPTION
**Pull request by gnardini**
### Description

In the WalletConnect integration we are returning the MTW address instead of the EOA. This is super dangerous! Users that connect using WalletConnect might end up making transactions to the MTW address and recovering the funds might be a bit painful (they wouldn't be lost, but it would be annoying).

### Other changes

N/A

### Tested

Using [use-contractkit](http://use-contractkit.vercel.app/)

### How others should test

Hard to test because [the public version of use-contractkit](http://use-contractkit.vercel.app/) doesn't work (I tested with a local version with some minor changes). If it did work one could simply connect your account in that website and check that the address listed is the EOA.

### Related issues

- Fixes #1075

### Backwards compatibility

N/A

### CODE REVIEW

File: packages/mobile/src/web3/saga.ts [hunk 1]
```diff
@@ -263,6 +263,8 @@ export function* getAccount() {
   }
 }
 
+export const getAccount = getWalletAddress
```
Comment (tarikbellamine) [inline packages/mobile/src/web3/saga.ts:266]: Can you add a comment here to explain that this is incorrect but requires a larger refactor to fix?

**Review by tarikbellamine — APPROVED** [review]

**Comment by ValoraQA** [comment]
Hey @gnardini , We have verified this issue using latest Test flight build V 1.19.0 and Android internal build V1.19.0 & updated status in Gsuite.

GSuite Link: https://docs.google.com/spreadsheets/d/1bWi6smdTMClreG96sGjYxb2lee9Ja6eT4-Lm7eXm1gM/edit#gid=761279479

Verified On Devices: Redmi note 8 (10.0), iPhone 12 (14.7.1)

Query:
We have observed that, we are not able to scan QR code using [use-contractkit](https://use-contractkit.vercel.app/)  on  latest Test flight build V 1.19.0 and Android internal build V1.19.0 from scan section of connected apps. **Attachment:** [Not Able to scan Qr Code.mp4](https://drive.google.com/file/d/1xhHeff3qnhLEB52PDgBpObaFJnQ2pFnJ/view?usp=sharing)

CC: @tarikbellamine

**Comment by MuckT** [comment]
https://valora.testquality.com requirement [R87](https://valora.testquality.com/project/12128/testcase/299603/requirement/11858).

**Comment by ValoraQA** [comment]
Hey @MuckT We have verified this issue using latest Test flight build V 1.19.0 and Android internal build V1.19.0 & updated status in Test Quality Tool

**Test Quality Tool link:
Android (9.0)**:   https://valora.testquality.com/project/12128/plan/21816/run/150277/result/2485120                                                     **Verified On Devices:** Redmi Note 8 (9.0)                                                                                                                                @gnardini

**Pull request merged**